src/renault_api/renault_vehicle.py

```diff
@@ -489,13 +489,30 @@ async def set_charge_mode(
     async def set_charge_start(self) -> models.KamereonVehicleChargingStartActionData:
         """Start vehicle charge."""
         # await self.warn_on_method("set_charge_start")
-        attributes = {"action": "start"}
-
         response = await self.session.set_vehicle_action(
             account_id=self.account_id,
             vin=self.vin,
             endpoint="charging-start",
-            attributes=attributes,
+            attributes={"action": "start"},
+        )
+        return cast(
+            models.KamereonVehicleChargingStartActionData,
+            response.get_attributes(
+                schemas.KamereonVehicleChargingStartActionDataSchema
+            ),
+        )
+
+    async def set_charge_pause_resume(
+        self, action: str
+    ) -> models.KamereonVehicleChargingStartActionData:
```

Comment:
Please rename to set_charge_stop. Then inside the function you check if `uses_endpoint_via_kcm`, and send the correct request (pause or raise NotImplemented).
```suggestion
    async def set_charge_stop(self) -> models.KamereonVehicleChargingStartActionData:
```

You can do the same in set_charge_start, which should check if `uses_endpoint_via_kcm`, and send the correct request (kca start, or kcm resume).